builder/vmware/common/run_config_test.go

```diff
@@ -0,0 +1,140 @@
+package common
```

Comment:
Do you think it would be a good idea to setup a mock vnc server to test connection and that the sent commands are actually sent ?

Reply:
I will have to set a WebSocket mock as the proxy for the vnc server mock to make sure the vnc over websocket connection steps are correct. I don't how yet how to that and if it's simple, but I'll give it a try.
I think for the VNC without websocket that wouldn't be necessary because go-vnc is responsible for connecting and sending the keyevents and I assume the library is already tested.